ui/src/apps/overview/RootComponent.js

```diff
@@ -42,7 +71,16 @@ const App = () => {
                 bordered={false}
                 title={t('overview.top_statements.title')}
               >
-                <Skeleton active />
+                {loadingStatements ? (
```

Comment:
Prefer to also add a redirection link to see statement details.
We also need to display the cover time of these statements.

Reply:
Clicking the SQL digest will jump to its detail page, here you mean to add a link to statement overview page?